### DISCUSSION TITLE
A small request about Sword Saint and Kineticist

### DISCUSSION THREAD

**Discussion by Danimeo224** [Truinto/DarkCodex#78] [Ideas]
Hi Truinto, 
If it wouldn't be too much trouble, could you please make Kinetic Blast an option for the Sword Saint's Chosen Weapon feature? The purpose is to let it scale with Kinetic Blade/ your Kinetic Whip Infusion.

Thanks in advance.

**Comment by Danimeo224** [comment]
Oh, I saw that one of the options was to let Sword Saint pick any weapons. I didn't see it on the mod description up here so I assumed it wasn't a thing. Thank you for adding it.

**Comment by Truinto** [comment]
I am not always remembering to update the list.

**Reply by Danimeo224** [reply]
That's alright, thank you for your hard work :)
